scripts/local-universe.py

```diff
@@ -115,38 +171,121 @@ def enumerate_docker_images(package_path):
     return (name for _, name in dockers.items())
 
 
-def download_docker_image(name, use_sudo):
+@contextlib.contextmanager
+def run_docker_registry(volume_path):
```

Comment:
The end result is a docker container that has a running docker registry (with all the containers pushed) and a running nginx serving up static files. I can't put `push` from a local docker to remote in a Dockerfile. So, the registry starts up with a local volume (`/tmp/foobar`). We then push all the images to the registry and shut it down. At this point in time, `/tmp/foobar` will have the layers and everything correctly laid out in it. These files then get placed into the end container via. a Dockerfile.